pkg/tsdb/azuremonitor/loganalytics/azure-response-table-frame_test.go

```diff
@@ -139,12 +139,27 @@ func TestLogTableToFrame(t *testing.T) {
 				return frame
 			},
 		},
+		{
+			name:     "data and error in real response",
+			testFile: "loganalytics/9-log-analytics-response-error.json",
+			expectedFrame: func() *data.Frame {
+				frame := data.NewFrame("",
+					data.NewField("OperationName", nil, []*string{pointer.String("Create or Update Virtual Machine")}),
+					data.NewField("Level", nil, []*string{pointer.String("Informational")}),
+				)
+				frame.Meta = &data.FrameMeta{
+					Custom:  &LogAnalyticsMeta{ColumnTypes: []string{"string", "string"}},
+					Notices: []data.Notice{{Severity: data.NoticeSeverityError, Text: "There were some errors when processing your query. Something went wrong processing your query on the server. The results of this query exceed the set limit of 1 records."}},
```

Comment:
Would you add a test documenting how a `data.NoticeSeverityWarning` level Notice would be returned? https://github.com/grafana/grafana/pull/51320/files#diff-a719bca7b751a8b6a6df86dac0bed3d54faf3a0282bf46407eb953803f303d04R17